tests/beta/test_default_tensor_tf.py

```diff
@@ -453,7 +453,7 @@ def test_jacobian_variable_multiply(self, torch_support, rep, tol):
 
         dev = qml.device("default.tensor.tf", wires=1, representation=rep)
 
-        @qml.qnode(dev)
+        @qml.qnode(dev, diff_method="parameter-shift")
```

Comment:
These test cases meant to use the parameter shift rule, but were now failing because `diff_method="best"` was set for a backdrop device.